battleGame/Intermediate/Config/CoalescedSourceConfigs/Engine.ini

```diff
@@ -1072,8 +1072,8 @@ MaxObjectsNotConsideredByGC=0
 SizeOfPermanentObjectPool=0
 AsyncIOBandwidthLimit=0
 Paths=../../../Engine/Content
```

Comment:
So this is going to be a potential problem.  I mean, we could just make sure all our directories structures are the same, but that might now be possible (especially if Brandon wants to code on his Mac).  This isn't throwing a merge conflict either.  We will have to think about this.